### DISCUSSION TITLE
Curation Report Week of May 10-16

### DISCUSSION THREAD

**Discussion by Finterly** [wikipathways/GPML-Repo#66] [Curation reports]
Dear admins,

There were 16 changed pathways and 10 pathways under construction. 

Question: there are two old pathways, are they still being worked on? 
WP2601 Gastric pepsin release (Homo sapiens)
WP2599 Gastric gastrin release (Homo sapiens) 
Both are created by User Andra (username) and last edited 2 December 2013?

_Summary_
Recent changes: 16
Under construction: 10
Stub pathways: 3

Next up: Tina

Fin

**Comment by egonw** [comment]
> Question: there are two old pathways, are they still being worked on? WP2601 Gastric pepsin release (Homo sapiens) WP2599 Gastric gastrin release (Homo sapiens) Both are created by User Andra (username) and last edited 2 December 2013?

I am talking to @andrawaag about these two pathways. To be continued soon.
